### PR TITLE
Dave/text carousel component landing page

### PR DESCRIPTION
**Pull request by MarkCarsonDev**
Redesigned TextCarousel component to incl. all items, transition smoothly, and have transparency mask to be more accurate to original design.

### CODE REVIEW

File: client/src/ui/TextCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
+"use client";
+import React, { useState, useEffect } from "react";
+import { Typography, Box } from "@mui/material";
+import "@/ui/styling/TextCarousel.css";
```
Comment (MarkCarsonDev) [inline client/src/ui/TextCarousel.tsx]: CSS components from dave go unused

File: client/src/ui/styling/TextCarousel.css
Comment (MarkCarsonDev) [inline client/src/ui/styling/TextCarousel.css:1]: This whole file goes unused. Remove.

File: client/src/ui/TextCarousel.tsx
Comment (MekhiHart) [inline client/src/ui/TextCarousel.tsx:1]: Nice job. However UI breaks for smaller screen (ipad: 1040x755)

image

**Review by MekhiHart — CHANGES_REQUESTED** [review]

File: client/src/ui/TextCarousel.tsx
Comment (MarkCarsonDev) [reply client/src/ui/TextCarousel.tsx:1]: thanks, will work on this, i think I know the issue

**Review by MekhiHart — APPROVED** [review]

**Pull request merged**